tests/network/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+@pytest.fixture(scope='package')
+def host_no_sdn_controller(host):
+    """ An XCP-ng with no SDN controller. """
+    if host.xe('sdn-controller-list', minimal=True) != '':
+        pytest.skip(f"This test requires an XCP-ng with no SDN controller")
```

Comment:
formulation makes me uneasy - wouldn't something like "conflicts with the presence of a SDN controller" be correct ?

Reply:
Why not:

```python
if host.xe('sdn-controller-list', minimal=True)
```

instead of:

```python
if host.xe('sdn-controller-list', minimal=True) != ''
```